ripe/atlas/tools/helpers/validators.py

```diff
@@ -47,25 +37,90 @@ def datetime(string):
                 "UTC."
             )
 
+    @staticmethod
+    def ip_or_domain(string):
+        message = '"{}" does not appear to be an IP address or host ' \
+                  'name'.format(string)
+
+        if " " in string:
+            raise argparse.ArgumentTypeError(message)
+        if "." not in string and ":" not in string:
+            if not re.match(r"^\w+$", string):
+                raise argparse.ArgumentTypeError(message)
+
+        return string
+
```

Comment:
I see where you're coming from on this, so I won't be adding more.  I just wanted to save the request/response turn-around trip for our users if I could.